CODIGO/frmMain.frm

```diff
@@ -3732,7 +3733,7 @@ Private Sub picInv_MouseMove(Button As Integer, Shift As Integer, x As Single, y
         Select Case ObjData(Inventario.ObjIndex(Slot)).ObjType
 
             Case eObjType.otWeapon
-                ObjLbl = Inventario.ItemName(Slot) & " (" & Inventario.Amount(Slot) & ")" & vbCrLf & "Daño: " & ObjData(Inventario.ObjIndex(Slot)).MinHit & "/" & ObjData(Inventario.ObjIndex(Slot)).MaxHit
+                ObjLbl = Inventario.ItemName(Slot) & " (" & Inventario.Amount(Slot) & ")" & vbCrLf & "Daño: " & ObjData(Inventario.OBJIndex(Slot)).MinHit & "/" & ObjData(Inventario.OBJIndex(Slot)).MaxHit
```

Comment:
Can you please move this out to a new module `language_utils` or something like that?

We must include the `english` and `spanish` strings.


Lets avoid using string literals among the code from now on

Reply:
I can but thats not really part of this pr, we should do it as dedicate effort for that

Reply:
Ok, agreed out of the scope of this patch.

Lets rework that later.